packages/firestore/test/util/helpers.ts

```diff
@@ -1073,3 +1074,40 @@ export function computeCombinations<T>(input: T[]): T[][] {
   };
   return computeNonEmptyCombinations(input).concat([[]]);
 }
+
+/**
+ * Helper method to generate bloom filter proto value for mocking watch
+ * filter response.
+ *
+ * If the return value is null, please increase the number of bits and/or
```

Comment:
I think this comment about `generateBloomFilterProto()` returning null is obsolete. Please remove it if that is the case.